docs/reference/query-dsl/geo-polygon-query.asciidoc

```diff
@@ -6,25 +6,29 @@ points. Here is an example:
 
 [source,js]
 --------------------------------------------------
+GET /_search
 {
-    "bool" : {
-        "query" : {
-            "match_all" : {}
-        },
-        "filter" : {
-            "geo_polygon" : {
-                "person.location" : {
-                    "points" : [
+    "query": {
+        "bool" : {
+            "must" : {
```

Comment:
Makes sense. Will do in a separate PR and apply the same modification to any other reference/query-dsl/geo* docs that have the same structure.